LICENSE-binary

```diff
@@ -208,52 +208,52 @@ License Version 2.0:
 audience-annotations-0.5.0
 commons-cli-1.4
 commons-lang3-3.8.1
-jackson-annotations-2.10.5
-jackson-core-2.10.5
-jackson-databind-2.10.5.1
-jackson-dataformat-csv-2.10.5
-jackson-datatype-jdk8-2.10.5
-jackson-jaxrs-base-2.10.5
-jackson-jaxrs-json-provider-2.10.5
-jackson-module-jaxb-annotations-2.10.5
+jackson-annotations-2.12.3
+jackson-core-2.12.3
+jackson-databind-2.12.3
+jackson-dataformat-csv-2.12.3
+jackson-datatype-jdk8-2.12.3
+jackson-jaxrs-base-2.12.3
+jackson-jaxrs-json-provider-2.12.3
+jackson-module-jaxb-annotations-2.12.3
 jackson-module-paranamer-2.10.5
-jackson-module-scala_2.13-2.10.5
+jackson-module-scala_2.13-2.12.3
 jakarta.validation-api-2.0.2
 javassist-3.27.0-GA
-jetty-client-9.4.38.v20210224
-jetty-continuation-9.4.38.v20210224
-jetty-http-9.4.38.v20210224
-jetty-io-9.4.38.v20210224
-jetty-security-9.4.38.v20210224
-jetty-server-9.4.38.v20210224
-jetty-servlet-9.4.38.v20210224
-jetty-servlets-9.4.38.v20210224
-jetty-util-9.4.38.v20210224
-jetty-util-ajax-9.4.38.v20210224
-jersey-common-2.31
-jersey-server-2.31
+jetty-client-9.4.43.v20210629
+jetty-continuation-9.4.43.v20210629
+jetty-http-9.4.43.v20210629
+jetty-io-9.4.43.v20210629
+jetty-security-9.4.43.v20210629
+jetty-server-9.4.43.v20210629
+jetty-servlet-9.4.43.v20210629
+jetty-servlets-9.4.43.v20210629
+jetty-util-9.4.43.v20210629
+jetty-util-ajax-9.4.43.v20210629
+jersey-common-2.34
+jersey-server-2.34
 log4j-1.2.17
 lz4-java-1.7.1
-maven-artifact-3.6.3
-metrics-core-2.2.0
-netty-buffer-4.1.59.Final
-netty-codec-4.1.59.Final
-netty-common-4.1.59.Final
-netty-handler-4.1.59.Final
-netty-resolver-4.1.59.Final
-netty-transport-4.1.59.Final
-netty-transport-native-epoll-4.1.59.Final
-netty-transport-native-unix-common-4.1.59.Final
+maven-artifact-3.8.1
+metrics-core-4.1.12.1
```

Comment:
This is a pretty big jump. How risky is this?

Reply:
I probably had to clarify. These updates are in the LICENSE file and reflect what we already ship with 3.0. 

They are not version upgrades themselves. So this PR bring the LICENSE file in sync to the actual versions we ship. In the process I discovered a single missing license. 

Apologies for the confusion.

Reply:
Similar to: https://github.com/apache/kafka/pull/10474

Reply:
The changes were autogenerated. I have a program that I'm hoping to share soon that checks for missing/stale licenses. It would correspond to: https://issues.apache.org/jira/browse/KAFKA-12622

Reply:
Ah, I see that now. Makes sense!